manifests/idesdk/pdk.pp

```diff
@@ -0,0 +1,35 @@
+# pdk.pp
+# Install Puppet Development Kit
+# https://puppet.com/download-puppet-development-kit
+#
+
+class software::idesdk::pdk (
+  $ensure  = $software::params::software_ensure,
+  $version = $software::params::pdk_version,
+  $url     = $software::params::pdk_url,
+) inherits software::params {
+
+  validate_string($ensure)
```

Comment:
Can we get rid of the legacy validate. I think I have just been removing this one from the other profiles.